src/renderer/components/Icons.tsx

```diff
@@ -1,9 +1,11 @@
-import { IoGrid } from 'react-icons/io5'
-import { MdClose, MdCreateNewFolder, MdGrid4X4, MdMoreVert } from 'react-icons/md'
+import { IoGrid, IoCloseCircleOutline } from 'react-icons/io5'
+import { MdCreateNewFolder, MdGrid4X4, MdMoreVert } from 'react-icons/md'
+
+
 
 export const AddFolderIcon = MdCreateNewFolder
 export const MenuIcon = MdMoreVert
-export const RemoveIcon = MdClose
+export const RemoveIcon = IoCloseCircleOutline
```

Comment:
I like the original icon better, I think this looks weird inside a circular button

Reply:
Yeah for sure